guides/common/modules/ref_storage-guidelines.adoc

```diff
@@ -24,12 +22,6 @@ endif::[]
 
 .File System Guidelines
 
-* Use the XFS file system for {ProjectNameX} because it does not have the inode limitations that `ext4` does.
-Because {ProductName} uses a lot of symbolic links it is likely that your system might run out of inodes if using `ext4` and the default number of inodes.
```

Comment:
Question for the Pulp folks, do you think this still holds true or could be removed?  Looks like a ~300GB file system would have ~18.7 million inodes, which i would think would be enough for just the content and generated metadata files.

Reply:
Yes pulp3 does not use inodes anymore for publishing content, so I think ext4 is a fine choice once again.